packages/jsonc/jsonc.test.mjs

```diff
@@ -0,0 +1,66 @@
+import assert from 'node:assert/strict';
+import { describe, it } from 'node:test';
+import { assertSuffixedSpecifiers } from '../../fixtures/assert-suffixed-specifiers.fixture.mjs';
+import { nextResolve } from '../../fixtures/nextResolve.fixture.mjs';
+import { nextLoad } from '../../fixtures/nextLoad.fixture.mjs';
+import { resolve, load } from './jsonc.mjs';
+
+describe('jsonc loader', { concurrency: true }, () => {
+	describe('resolve', () => {
+		it('should recognise jsonc files', async () => {
+			const result = await resolve('./fixture.jsonc', { importAttributes: { type: 'jsonc' } }, nextResolve);
+
+			assert.deepEqual(result, {
+				format: 'jsonc',
+				url: './fixture.jsonc',
+			});
+		});
```

Comment:
Is an import attribute required to import jsonc?

Also, I think maybe have 2 additional test cases:

1. file extension is something else: `resolve('./fixture.ext', { importAttributes: { type: 'jsonc' } }, nextResolve);`
2. import attributes is something else: `resolve('./fixture.jsonc', { importAttributes: { type: 'json' } }, nextResolve);`

Reply:
> Is an import attribute required to import jsonc?

iMO, yes because for JSON it's required

Reply:
Sorry, by "required", I meant "will node barf without it".